editors/code/src/ctx.ts

```diff
@@ -204,13 +204,13 @@ export class Ctx {
         }
     }
 
-    setServerStatus(status: ServerStatusParams) {
+    setServerStatus(status: ServerStatusParams | { health: "stopped" }) {
         let icon = "";
         const statusBar = this.statusBar;
         switch (status.health) {
             case "ok":
-                statusBar.tooltip = status.message ?? "Ready";
-                statusBar.command = undefined;
+                statusBar.tooltip = (status.message ?? "Ready") + "Click to stop.";
```

Comment:
There should probably be a space here.

Reply:
Saw that earlier as well, fixed in https://github.com/rust-lang/rust-analyzer/pull/13512